MergerLogic/Clients/HeartbeatClient.cs

```diff
@@ -44,13 +49,28 @@ public void Start(string taskId)
 
         public void Stop()
         {
-            if (!this._timer.Enabled)
+            try
             {
-                throw new Exception($"[{MethodBase.GetCurrentMethod().Name}] Heartbeat interval must be running in order to stop it.");
+                string relativeUri = $"heartbeat/remove";
```

Comment:
Move the remove heartbeat after timer stopped

Reply:
fixed